discord/raw_models.py

```diff
@@ -185,15 +185,21 @@ class RawMessageUpdateEvent(_RawReprMixin):
     cached_message: Optional[:class:`Message`]
         The cached message, if found in the internal message cache. Represents the message before
         it is modified by the data in :attr:`RawMessageUpdateEvent.data`.
+    new_message: Optional[:class:`Message`]
```

Comment:
Why is this optional? The parameter type doesn't allow it to be, and it shouldn't.
```suggestion
    new_message: :class:`Message`
```